sktime/forecasting/base/_base.py

```diff
@@ -914,6 +912,7 @@ def _update_predict_single(
         fh,
         X=None,
         update_params=True,
+        # todo will be deprecated after refactor:
```

Comment:
Can we add a `TODO: remove in v0.9.0` comment to the lines that we want to remove? Perhaps also add that info to the deprecation messages.